helpers/constants.js

```diff
@@ -135,6 +135,7 @@ var constants = {
 	// WARNING: When changing totalAmount you also need to change getBlockRewards(int) SQL function!
 	totalAmount: 10000000000000000,
 	unconfirmedTransactionTimeOut: 10800, // 1080 blocks
+	PUBLIC_KEY_HEX_LENGTH: 64,
```

Comment:
Imo, this should be named `publicKeyHexLength` according to other properties in this object. Deviating from this notation style creates an inconsistency.